### PR TITLE
#1254 Small bug fixes and translation improvements

### PR DESCRIPTION
**Pull request by stephanevieira75**

### CODE REVIEW

File: twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx [hunk 1]
```diff
@@ -76,7 +76,8 @@ export default (props: Props) => {
           } else if (app.display?.messages_module?.right_icon) {
             addon_right_icon.push(app);
           } else {
-            addon_menu.push(menu_item);
+            // Not sure about this, giphy still displayed on web.qa.twake
+            app.simple_name !== 'giphy' && addon_menu.push(menu_item);
```
Comment (stephanevieira75) [inline twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx:80]: @RomaricMourgues Could you check if this works on QA please? I'm not sure about this.

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx:80]: No no, please revert this, removing Giphy is something we do in administration not in code :)

Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx:80]: What did you try to do exactly ? @stephanevieira75

File: twake/frontend/public/locales/fr.json [hunk 2]
```diff
@@ -1655,5 +1655,6 @@
     "{{$1}} - Vous ne pouvez pas ajouter plus de {{$2}} fichiers dans un message",
     "scenes.apps.messages.message.parts.deleted_content.text.current_user":
     "Vous avez supprimé ce message",
-    "scenes.apps.messages.message.parts.deleted_content.text": "{{$1}} a supprimé son message"
+    "scenes.apps.messages.message.parts.deleted_content.text": "{{$1}} a supprimé son message",
+    "general.uploading": "Téléchargement"
```
Comment (RomaricMourgues) [inline twake/frontend/public/locales/fr.json:1659]: Isn't it "Envoi" instead ?

**Comment by RomaricMourgues** [comment]
@stephanevieira75 2 changes please, if you can do them quickly great, if not I will do it this weekend myself :)

File: twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx (hunk 1 above)
Comment (RomaricMourgues) [reply twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx:80]: If the goal was "remove giphy" then just don't do it by code, I will handle it in the configuration of Twake directly.

Comment (stephanevieira75) [reply twake/frontend/src/app/scenes/Apps/Messages/Input/Parts/InputOptions.tsx]: Removed 👍🏻

File: twake/frontend/public/locales/fr.json (hunk 2 above)
Comment (stephanevieira75) [reply twake/frontend/public/locales/fr.json]: Updated 👍🏻

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**